platform/marvell/sai.mk

```diff
@@ -4,7 +4,7 @@ BRANCH = 202405
 ifeq ($(CONFIGURED_ARCH),arm64)
 MRVL_SAI_VERSION = 1.14.0-3
 else ifeq ($(CONFIGURED_ARCH),armhf)
-MRVL_SAI_VERSION = 1.14.0-2
+MRVL_SAI_VERSION = 1.14.0-3
```

Comment:
Has this version merged to 202405 branch?

Reply:
This PR is targeted for 202405.